conda_env.yml

```diff
@@ -1,5 +1,6 @@
 name: stream
 channels:
+- file://home/1116449/miniconda2/custom_conda_channel
```

Comment:
What file is this? Doesn't seem it would be available to the Travis server.

Reply:
This should be removed. It is a holdover from my local testing. Because the supporting (QARTOD) versions of ion-functions and ooi-data are not available on anaconda yet, I have to install them from a local channel. That line is an artifact of that process. I'll revert it.